nextflow.config

```
@@ -9,13 +9,50 @@
 // Global default params, used in configs
 params {
 
-    // TODO nf-core: Specify your pipeline's command line flags
     // Input options
     input                      = null
-    // References
-    genome                     = null
-    igenomes_base              = 's3://ngi-igenomes/igenomes/'
-    igenomes_ignore            = false
+    transcript_fasta           = null
+
+    // Workflow options
+    qc_only                    = false
+    skip_assembly              = false
+
+    // Trimming options
+    adapter_fasta              = null
+    save_trimmed_fail          = false
+    save_merged                = false
+    skip_fastp                 = false
+    skip_fastqc                = false
+    extra_fastp_args           = null
+
+    // SortMeRNA options
+    remove_ribo_rna            = false
+    save_non_ribo_reads        = false
+    ribo_database_manifest     = "${projectDir}/assets/rrna-db-defaults.txt"
+
+    // Assembly options
+    assemblers                 = "trinity,rnaspades"
+    extra_trinity_args         = null // applies to both trinity and trinity_no_norm
+    soft_filtered_transcripts  = false
+    hard_filtered_transcripts  = false
+    ss                         = null
```

```suggestion
    strand_specific             = null
```
if you agree with my comment before

---

same as before (param is from rnaSPAdes)